configure.ac

```diff
@@ -211,9 +212,10 @@ AS_IF([test "$use_doxygen" != "no"],
 AC_SYS_LARGEFILE
 
 
-# fish does not use exceptions
+# Fish does not use exceptions itself. However the MuParser library does so we
+# need to enable support for them and pay the cost of larger code. Note:
 # Disabling exceptions saves about 20% (!) of the compiled code size
-CXXFLAGS="$CXXFLAGS -fno-exceptions"
+#CXXFLAGS="$CXXFLAGS -fcxx-exceptions"
```

Comment:
The good news is that the comment is no longer true, at least on my system: with exceptions and debug information stripped, I get a 1.4 M fish executable; without exceptions produces a 1.3 M executable. (g++ 4.9)